Dockerfile

```diff
@@ -18,6 +18,9 @@ ENV DOCKER_DD_AGENT=yes \
     DD_SUPERVISOR_DELETE_USER=yes \
     DD_CONF_PROCFS_PATH="/host/proc"
 
+# workaround for stretch-slim missing man dirs
+RUN seq 1 8 | xargs -I{} mkdir -p /usr/share/man/man{}
```

Comment:
I'm curious, why is this necessary?

Reply:
I actually have to check if it's still necessary with `buster` - on slim those directories are missing, I left it in there because it's a NOP if the directory structure is in place. Not having it in place was breaking the build with `stretch-slim`, it was a well documented issue.